industry_benchmarks/utils/data_gathering.py

```diff
@@ -490,26 +772,54 @@ def gather_data(
     output_dir.mkdir(exist_ok=False, parents=True)
 
     # GATHER Input based information
-    ligand_network = parse_ligand_network(input_ligand_network)
+    alchemical_network = parse_alchemical_network(input_alchemical_network)
+    ligand_network = extract_ligand_network(alchemical_network)
     # Combine old + new LigandNetwork if a fixed network is provided
-    if fixed_ligand_network:
-        fixed_network = parse_ligand_network(fixed_ligand_network)
+    if fixed_alchemical_network is not None:
+        fixed_alchemical_network = parse_alchemical_network(fixed_alchemical_network)
+        fixed_network = extract_ligand_network(fixed_alchemical_network)
         ligand_network = ligand_network.enlarge_graph(
             edges=fixed_network.edges, nodes=fixed_network.nodes)
+        # combine the alchemical networks
+        alchemical_network = gufe.AlchemicalNetwork(edges=[*alchemical_network.edges, *fixed_alchemical_network.edges])
+
+    # get the ligand and edge scores
     transformation_scores = gather_transformation_scores(ligand_network)
     ligand_scores = gather_ligand_scores(ligand_network)
+
+    # process the results one by one
+    collected_results = process_results(results_folder, output_dir, alchemical_network)
+
+    failed_transforms = []
+    # workout which edges must have failed
+    for name, results in collected_results.items():
```

Comment:
`collected_results` is the `estimates` dict from above where the results for the individual repeats would under different keys if I understand it correctly. So would the `len(results)` always be one or am I missing something?

Reply:
Changed the layout a bit for this which should make it clearer.